src/arrays/ArrayUtil.java

```diff
@@ -1,23 +1,37 @@
 package arrays;
 
 
+import java.util.Arrays;
+
 /**
  * An utility class containing methods for convenient work with arrays
  */
 public class ArrayUtil {
 
-
     /**
      * Create an array with the given size, fill it with the different values and print it.
      */
     public static void createArray(int size) {
-
+        int[] array = new int[]{1, 2, 3, 4, 5, 6, 7, 8, 9, 10};{
```

Comment:
If your method has arguments, you should definitely use it.
here in the array you can add size and get random values.
You can write for example like this
    int[] array = new int[size];
        for (int i = 0; i < array.length; i++) {
            array[i] = (int) (Math.random() * 100);
            System.out.print(array[i] + " ");

        }